### ISSUE TITLE
[MagicNumber] Allow ignoring default values in constructor properties

### ISSUE THREAD

**Issue by arturbosch** [arturbosch/detekt#643]
As mentioned in [sonar-kotlin](https://github.com/arturbosch/sonar-kotlin/issues/49#issuecomment-353542268)
the default values are always reported: `class SomeClassWithDefault(val defaultValue: Int = 10)`

Maybe we should also add an ignore property for class/object constructor properties.

Here a first test case:
```kotlin
given("in class declaration with default properties") {
    val ktFile = compileContentForTest("class SomeClassWithDefault(val defaultValue: Int = 10)")

    it("should report no finding") {
        assertThat(MagicNumber().lint(ktFile)).isEmpty()
    }
}
```

**Comment by schalkms** [comment]
👌

**Comment by schalkms** [comment]
Closed in 07555cf
